### ISSUE TITLE
method for simultaneous insert into master and part tables?

### ISSUE THREAD

**Issue by neuralsignal** [datajoint/datajoint-python#711]
I have implemented a utility method (called insert1p) on my branch that inserts a single master entry both into a master and its part tables: https://github.com/gucky92/datajoint-python/commit/b3d037ecd24c7184c0d4c64613f7bd7ef6e2831f 

One can pass either a dictionary, a recarray, or a pandas dataframe. If it is a recarray or pandas dataframe, it internally checks if the array only consists of one unique entry for the master. But it does allow for multiple entries into the part tables. 
Is there an interest in this type of method?

The same commit also allows implements a property for tables that returns a list of the part table classes belonging to this master.

**Comment by dimitri-yatsenko** [comment]
Hi @gucky92 I have reviewed your proposed feature. More generally, would it make more sense to implement `insert` on the `Join` class. It would look something like:
```
(Session * Scan * Behavior).insert(df)
```
This would cover the case for master-part tables. You would need to specify them all explicitly but arguably that is a desired property.

**Comment by gucky92** [comment]
Yes, that would be the more general case. In that case, it may be best to write an insert mixin class or class that gets called similarly as in the case of fetch. Do you agree?

**Comment by yambottle** [comment]
This issue is closed because there has been no activity for more than 365 days, please reopen if it's still applicable.
